backend/src/test/groovy/com/capstone/moneytree/controller/UserControllerTest.groovy

```diff
@@ -235,4 +247,267 @@ class UserControllerTest extends Specification {
       then: "Should throw CredentialNotFoundException"
       thrown(CredentialNotFoundException)
    }
+
+//   @Test
```

Comment:
Were these test commented out because they were failing?

Reply:
No this was the old test, I updated all new test below with more test cases and coverage, I need to remove this commented code, I will update the PR. Cant believe I missed that